delayScanPlots.py

```diff
@@ -1,48 +1,81 @@
 import numpy as np
 import matplotlib.pyplot as plt
 import matplotlib.patches as mpatches
-
+import sys
 import matplotlib.colors as mcolors
+import pandas as pd
+import mplhep as hep
+plt.style.use(hep.style.CMS)
+plt.rcParams.update({'font.size': 18})
+import json
+import glob
+import pandas as pd
+import mplhep as hep
+plt.style.use(hep.style.CMS)
+plt.rcParams.update({'font.size': 18})
+import json
+import glob
 
-import mplhep
-mplhep.style.use(mplhep.style.CMS)
-
-
-def delay_scan_plots(fname,
-                     title='ETx Delay Scan',
-                     outputFileName=None,
-                     ECOND=False):
-
-    data = np.load(f'./{fname}.npz',allow_pickle=True )
-    x = data['errorcounts'].flatten()[0]
-    y = data['bitcounts'].flatten()[0]
-    errorRates = []
-
-    for i in range(6):
-       errorRates.append(list(np.array(x[i])/np.array(y[i])))
-
-    errorRates = np.array(errorRates).T.flatten()
-
-    if ECOND:
-        a,b=np.meshgrid(np.arange(6),np.arange(63))
-    else:
-        a,b=np.meshgrid(np.arange(13),np.arange(63))
 
-    fig,ax=plt.subplots()
+def jsonload(fname):
+    with open(fname) as jsonfile:
+        try:
+            return json.load(jsonfile)
+        except Exception:
+            print(fname)
 
-    plt.hist2d(a.flatten(),b.flatten(),
-               weights=errorRates,
-               bins=(np.arange(7)-0.5,np.arange(64)-0.5),
-               cmap='RdYlBu_r',
-               alpha=errorRates>0,
-               figure=fig);
 
-    plt.colorbar().set_label(label='Transmission errors rate ',size=32)
-    plt.title(title,size=32)
-    plt.ylabel('Phase Select Setting', size=32)
-    plt.xlabel('Channel Number', size=32)
-    if outputFileName:
-        plt.savefig(f'./ECOND_delay_scan_board{boardNum}_{freq}.png',dpi=300, facecolor = "w")
-    plt.close(fig)
+def json_delayscan(indir = ""):
+    fdir = f"{indir}"
```

Comment:
What's the point of this line?  either rename the function argument to `fdir` or use `indir` in the next line.